pkgs/applications/science/logic/key/default.nix

```diff
@@ -38,7 +38,8 @@ let
         | sh
     '';
     outputHashMode = "recursive";
-    outputHash = "sha256-1TWySkS8w7L6Q+V946kcLOnM4hL3fieFvLrF5BZAlh4=";
+    outputHashAlgo = "sha256";
+    outputHash = "sha256-GjBUwJxeyJA6vGrPQVtNpcHb4CJlNlY4kHt1PT21xjo=";
```

Comment:
isn't "sha256" already implied by the sri hash?

Reply:
Seems reasonable to me. I'll merge and we can always take alternative approaches in a follow-up.

Reply:
> Is there a quicker way to do this?

I usually change a single character in the hash or replace it with `lib.fakeHash`

Reply:
Thanks for the tip. I'll use `lib.fakeHash` in the future.